ENVIRONMENTVARS.md

```diff
@@ -0,0 +1,59 @@
+| -------- | ---- | ------------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `NYPL_HEADER_URL` | string | "https://ds-header.nypl.org" | The base URL of the NYPL envionment-specific header and footer scripts.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+
+## AWS Elastic Beanstalk Environment Variables
+
+As previously mentioned in the [README](README.md), we are using environment variables to make authorized requests to NYPL's API platform. In order to be secure, we are encrypting and decrypting those environment variables using AWS KMS. Please get these variables from someone in the NYPL Digital Department.
+
+| Variable | Description |
+| -------- | ----------- |
+| `KMS_ENV` | Determines whether to interpret ..CLIENT_ID and ..CLIENT_SECRET variables as "encrypted" or "unencrypted". Default "encrypted". | 
+| `PLATFORM_API_CLIENT_ID` | Platform client id. If KMS_ENV is "encrypted", this value must be encrypted. |
+| `PLATFORM_API_CLIENT_SECRET` | Platform client secret. If KMS_ENV is "encrypted", this value must be encrypted. |
+
+### Encrypting
```

Comment:
have you met my friend [kms-util](https://github.com/NYPL-discovery/kms-util)?

Reply:
Good to know about kms-util! Not sure if we need to support unencrypted vars, unsure if there's a historical reason for this but happy to remove the support for unencrypted vars if everyone agrees.

Reply:
Wow `kms-utils` is very useful. Should that be moved to the `NYPL` Github org (eventually)?

My initial thought is to support unencrypted env vars for local development if a dev doesn't have AWS set up locally or have the encrypted vars. But this doesn't really make sense and it seems fair to say that a dev working on this app _must_ have AWS access to nypl-digital-dev and be logged in locally. Thoughts?

Reply:
I agree with that, Edwin! Supporting unencrypted vars also runs a higher risk of accidentally committing secrets.

Reply:
Good call! removed the ability to use unencrypted credentials and updated the docs.